app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  validates :username, presence: true
+  has_many :users_events
+  has_many :attended_events, through: :users_events
+  has_many :created_events, foreign_key: 'user_id', class_name: 'Event'
```

Comment:
**A suggestion:** Using the suggested `creator_id` column in the Event Model, the association could be: `has_many :events, foreign_key: :creator_id`
That would make your code easier to understand and you wouldn't need to specify the `class_name`

Reply:
We purposely named our association `created_events` as that was what was required of us in the project specification.